HerPortal/Constants.cs

```diff
@@ -3,4 +3,7 @@
 public static class Constants
 {
     public const string SERVICE_NAME = "Service name";
-}
\ No newline at end of file
+
+    // This must be kept in sync with the corresponding variable in app.scss
+    public const string BASE_PATH = "/portal";
```

Comment:
This can obviously be easily changed...

I wanted to make it configuration rather than a constant, but it needs to be known statically during the npm build, so there's limited benefit.

(We could template it dynamically when serving the assets, but that seems unlikely to be worthwhile)